### PR TITLE
MINOR: Add logging to ImageThumbnail task

### PR DESCRIPTION
**Pull request by unclecheese**
Related: https://github.com/silverstripe/silverstripe-assets/pull/251/files

### CODE REVIEW

File: code/Helper/ImageThumbnailHelper.php [hunk 1]
```diff
@@ -12,11 +13,26 @@ class ImageThumbnailHelper
 {
     use Injectable;
 
+    private static $dependencies = [
+        'logger' => '%$' . LoggerInterface::class . '.quiet',
+    ];
+
+    /** @var LoggerInterface|null */
```
Comment (robbieaverill) [inline code/Helper/ImageThumbnailHelper.php]: For property doc blocks it's probably nicer to expand this (minor thing though)

File: code/Helper/ImageThumbnailHelper.php [hunk 2]
```diff
@@ -12,11 +13,26 @@ class ImageThumbnailHelper
...
      * @var int
      */
     private $maxImageFileSize;
 
+    /**
+     * @param LoggerInterface $logger
+     */
+    public function setLogger(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
```
Comment (robbieaverill) [inline code/Helper/ImageThumbnailHelper.php]:
```suggestion
        $this->logger = $logger;
        return $this;
```

File: code/Helper/ImageThumbnailHelper.php [hunk 3]
```diff
@@ -12,11 +13,26 @@ class ImageThumbnailHelper
...
+    /** @var LoggerInterface|null */
+    private $logger;
+
     /**
      * @var int
      */
     private $maxImageFileSize;
 
+    /**
+     * @param LoggerInterface $logger
```
Comment (robbieaverill) [inline code/Helper/ImageThumbnailHelper.php]:
```suggestion
     * @param LoggerInterface $logger
     * @return $this
```

File: code/Helper/ImageThumbnailHelper.php (hunk 2 above)
Comment (unclecheese) [reply code/Helper/ImageThumbnailHelper.php]: Have updated.

**Comment by unclecheese** [comment]
Behat failures were preexisting.

**Pull request merged**